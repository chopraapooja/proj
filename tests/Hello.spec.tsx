import 'jest'
import * as React from "react";
import { mount }            from 'enzyme';
import toJSON from 'enzyme-to-json';

import { Hello } from '../src/components/Hello';

test("Dummy test", () => {
  const component = mount(<Hello compiler="TS" framework="React" />);
  expect(toJSON(component)).toMatchSnapshot();
})