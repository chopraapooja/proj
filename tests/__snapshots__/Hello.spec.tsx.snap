// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dummy test 1`] = `
<Component
  compiler="TS"
  framework="React"
>
  <h1>
    Hello World! from 
    TS
     and 
    React
    !
  </h1>
</Component>
`;
